File: Notes/app/src/main/java/com/example/notes/test/control/types/RequestType.java
package com.example.notes.test.control.types;

public enum RequestType {

    REQ_AUTHORIZE("REQ_AUTHORIZE", 1),
    REQ_REGISTER("REQ_REGISTER", 2),
    REQ_BIOMETRIC_LOGIN("REQ_BIOMETRIC_LOGIN", 3),
    REQ_UNLOCK_KEYSTORE("REQ_BIOMETRIC_LOGIN", 4);

    String description;
    int type;

    RequestType(String description, int type) {
        this.description = description;
        this.type = type;
    }

    public String getDescription() {
        return description;
    }

    public int getType() {
        return type;
    }
}

File: Notes/app/src/main/java/com/serhii/apps/notes/control/types/RequestType.kt
/*
 * Copyright 2022. Happy coding ! :)
 * Author: Serhii Butryk
 */
package com.serhii.apps.notes.control.types

enum class RequestType(var description: String, var type: Int) {
    REQ_AUTHORIZE("REQ_AUTHORIZE", 1), REQ_REGISTER(
        "REQ_REGISTER",
        2
    ),
    REQ_BIOMETRIC_LOGIN("REQ_BIOMETRIC_LOGIN", 3);
}